MediaBrowser.Providers/Plugins/Tmdb/TmdbClientManager.cs

```diff
@@ -140,49 +140,23 @@ public async Task<TvShow> GetSeriesAsync(int tmdbId, string language, string ima
         /// Gets a tv show episode group from the TMDb API based on the show id and the display order.
         /// </summary>
         /// <param name="tvShowId">The tv show's TMDb id.</param>
-        /// <param name="displayOrder">The display order.</param>
+        /// <param name="episodeGroupId">The Episode Group id.</param>
         /// <param name="language">The tv show's language.</param>
         /// <param name="imageLanguages">A comma-separated list of image languages.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The TMDb tv show episode group information or null if not found.</returns>
-        private async Task<TvGroupCollection> GetSeriesGroupAsync(int tvShowId, string displayOrder, string language, string imageLanguages, CancellationToken cancellationToken)
+        private async Task<TvGroupCollection> GetSeriesGroupAsync(int tvShowId, string episodeGroupId, string language, string imageLanguages, CancellationToken cancellationToken)
```

Comment:
episodeGroupId is a bit misleading for its type. We're utilizing ints for any form of ID throughout, but then are suddenly swapping to a string. We may want to consider renaming this.

Reply:
Hmm, would it be reasonable to slap "String" at the end?